### DISCUSSION TITLE
Feature request -- Jaro/Jaro-Winkler string distance

### DISCUSSION THREAD

**Discussion by ericmanning** [duckdb/duckdb#4194] [Ideas]
Text function feature request: Jaro-Winkler string distance metric with option for prefix weight. This metric is commonly used for fuzzy string matching. It can perform better than alternative string distance metrics (e.g., Levenshtein and Jaccard, which are already implemented in duckdb) for probabilistic record linkage with names and addresses.

Example implementation in C++ at [maxbachmann/JaroWinkler](https://github.com/maxbachmann/JaroWinkler).

**Comment by Mytherin** [comment]
This seems like a relatively straightforward function to add. Happy to review a PR, otherwise this would also be a good first issue.
